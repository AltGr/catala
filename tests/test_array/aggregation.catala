@Article@

/*
new scope A:
   param x content set money 

scope A:
   def x := [$0; $4 +$ $5; $8 *$ 0.65]

new scope B:
   param a scope A 
   param y content money
   param z content int

scope B:
   def y := sum money for m in a.x of (m +$ $1)
   def z := number for m in a.x of (m >=$ $8.95)
*/