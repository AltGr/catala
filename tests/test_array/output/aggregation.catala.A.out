catala: too many arguments, don't know what to do with `test_array/good/aggregation_2.catala'
Usage: catala [OPTION]... BACKEND FILE
Try `catala --help' for more information.
